test/ui/test.js

```diff
@@ -105,6 +108,33 @@ test.describe('UI tests', function() {
         .then(screenshotAndCompare('body.png', 'body'));
     });
 
+    test.it('shows(hides) extra buttons when a tree item is (un)selected', function() {
+      // simulate a list reload by calling the Jupyter function, and waiting
```

Comment:
I think block comments are a bit easier to read when they use normal sentence case and punctuation.